### ISSUE TITLE
Minimization over multiple variables

### ISSUE THREAD

**Issue by yig** [iheartla/iheartla#107]
I would like to be able to write a minimization expression over multiple variables simultaneously. I propose commas to separate minimization variables. For example, I would like the following to work:

```
given
x_i ∈ ℝ^n: Locations at which we have observations.
y_i ∈ ℝ^m: The corresponding observed values for x_i.

min_(R ∈ ℝ^(m×n), t ∈ ℝ^m) ∑_i ‖ Rx_i + t - y_i ‖²
```

[Open in Compiler](https://iheartla.github.io/iheartla/?code=given%0Ax_i+%E2%88%88+%E2%84%9D%5En%3A+Locations+at+which+we+have+observations.%0Ay_i+%E2%88%88+%E2%84%9D%5Em%3A+The+corresponding+observed+values+for+x_i.%0A%0Amin_%28R+%E2%88%88+%E2%84%9D%5E%28m%C3%97n%29%2C+t+%E2%88%88+%E2%84%9D%5Em%29+%E2%88%91_i+%E2%80%96+Rx_i+%2B+t+-+y_i+%E2%80%96%C2%B2%0A)

**Comment by alecjacobson** [comment]
Is this supported now?

**Comment by pressureless** [comment]
Yes, it's supported on `local_func` branch and only Numpy works now
